routes/projects.js

```diff
@@ -0,0 +1,16 @@
+import express from 'express';
+import projectsInfo from '../projectsInfo.js';
+
+const router = express.Router();
+
+router.get('/', (req, res) => {
+  res.send(projectsInfo);
+});
+
+router.get('/:text', (req, res) => {

```

Comment:
None of your routes has try-catch or any fail-safe mechanism. Read about `asyncMiddleware` that wraps everything into Promise, write error catcher and 404 response route.